cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinSecondaryCategory.java

```diff
@@ -11,6 +11,7 @@
 
 public enum CristinSecondaryCategory {
```

Comment:
remove the comments?

Reply:
I would like to keep those for further mapping of other types of cristin-results.
It is a bit time consuming to figure out wich cristin category contains what objects.